codegenerator/cli/templates/dynamic/codegen/src/Types.res.hbs

```diff
@@ -118,46 +107,27 @@ module Block = {
   let schema = {{field_selection.block_schema}}
 
   let rawEventSchema = {{field_selection.block_raw_event_schema}}
-
   {{#if is_evm_ecosystem}}
-  @get
-  external getNumber: t => int = "number"
-
-  @get
-  external getInternalNumber: Internal.eventBlock => int = "number"
 
   @get
-  external getTimestamp: t => int = "timestamp"
+  external getNumber: Internal.eventBlock => int = "number"
 
   @get
-  external getInternalTimestamp: Internal.eventBlock => int = "timestamp"
+  external getTimestamp: Internal.eventBlock => int = "timestamp"
  
   @get
-  external getId: t => string = "hash"
-
-  {{!-- TODO: Remove this by moving block hash to eventItem --}}
-  @get
-  external getInternalId: Internal.eventBlock => string = "hash"
+  external getId: Internal.eventBlock => string = "hash"
   {{/if}}
-
   {{#if is_fuel_ecosystem}}
-  @get
-  external getNumber: t => int = "height"
 
   @get
-  external getInternalNumber: Internal.eventBlock => int = "height"
+  external getNumber: Internal.eventBlock => int = "height"
```

Comment:
Is this binding correct? Looks wrong

Reply:
I don't see what's wrong

Reply:
The field "height" vs "number". Just wanted to check that it's intentional. The binding shouldn't change to "number" or getHeight?

Reply:
It should have the `getNumber` name to be compatible with the evm version. This specific code needed for compatibility between evm and fuel, because they have different field names. e.g height for block number